<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modificar_moto</class>
 <widget class="QDialog" name="Modificar_moto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QComboBox" name="cb_modificar_moto">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Modificar Moto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Placa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Km Recorridos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado de la  carretila para pasajero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_agragar_moto_cancelar_modificar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_reparado_moto_modificar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reparado</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="le_marca_moto_modificar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_carretilla_si_modificar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_estado_bueno_moto_modificar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bueno</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio de Compra:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gastos Reparacion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Compra carretilla</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_gastos_reparacion_modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_bueno_carretilla_modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bueno</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_estado_malo_moto_modificar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Malo</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gastos Reparacion carretilla</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_precio_compra_modificar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_agregar_moto_aceptar_modificar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_carretilla_no_modificar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_gastos_reparacion_carretilla_modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_km_recorridos_modificar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_malo_carretilla_modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Malo</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_estado_reparado_carretilla_modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reparado</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trae Carretila para pasajero?</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_precio_compra_carretilla_modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado de l a Moto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_placa_moto_modificar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_agragar_moto_cancelar_modificar</sender>
   <signal>clicked()</signal>
   <receiver>Modificar_moto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
